libraries/botframework-schema/src/teams/index.ts

```diff
@@ -1225,6 +1229,167 @@ export interface FileConsentCardResponse {
     uploadInfo?: FileUploadInfo;
 }
 
+/**
+ * @interface
+ * Current tab request context, i.e., the current theme.
+ */
+export interface TabContext {
+    /**
+     * @member {theme} [string] The current user's theme.
+     */
+    theme?: string;
+}
+
+/**
+ * @interface
+ * Current TabRequest entity context, or 'tabEntityId'.
+ *
+ */
+export interface TabEntityContext {
+    /**
+     * @member {string} [tabEntityId] The entity id of the tab.
+     */
+    tabEntityId?: string;
+}
+
+/**
+ * @interface
+ * Invoke ('tab/fetch') request value payload.
+ *
+ */
+export interface TabRequest {
+    /**
+     * @member {TabEntityContext} [tabContext] The current tab entity request context.
+     */
+    tabContext?: TabEntityContext;
+    /**
+     * @member {TabContext} [context] The current user context, i.e., the current theme.
+     */
+    context?: TabContext;
+    /**
+     * @member {string} [state] The magic code for OAuth flow.
+     */
+    state?: string;
+}
+
+/**
+ * @interface
+ * Envelope for Card Tab Response Payload.
+ *
+ */
+export interface TabResponse {
+    /**
+     * @member {TabResponsePayload} [tab] The response to the tab/fetch message.
+     */
+    tab?: TabResponsePayload;
+}
+
+/**
+ * @interface
+ * Envelope for cards for a Tab request.
+ *
+ */
+export interface TabResponseCard {
+    /**
+     * @member {object} [card] The adaptive card for this card tab response.
+     */
+    card?: Record<string, unknown>;
+}
+
+/**
+ * @interface
+ * Envelope for cards for a TabResponse.
+ *
+ */
+export interface TabResponseCards {
+    /**
+     * @member {object[]} [cards] Adaptive cards for this card tab response.
+     */
+    cards?: Record<string, unknown>[];
+}
+
```

Comment:
Is `{object[]}` appropriate with `Record<string, unknown>[]`? JSDoc doesn't seem to align with Typescript 100% and was giving me some yellow squigglies, which I tried to minimize.

Reply:
I've successfully used `Record` in JSDoc comments before. It's like the `unknown` bit throwing off JSDoc. Use `any` instead, but only in JSDoc comments 👍

Reply:
K. I'll make the changes to use Record in JSDoc